### PR TITLE
Add in example custom rule: code sample checker

### PR DESCRIPTION
**Pull request by sanscontext**
@lornajane 👋🏻 Hello from the WtD Slack!

### CODE REVIEW

**Review by lornajane — APPROVED** [review]
Thank you very much!

File: custom-plugin-rules/code-sample-checks/README.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+Save this file and import the plugin by adding the following example to `redocly.yaml`:
+
+```yaml
+plugins:
+- ./x-code-sample-checks.js
+```
+
+### Rule - Check SDK Coverage
+
+The main functionality of the plugin is in this file `check-dks-coverage.js`, which is as follows:
```
Comment (adamaltman) [inline custom-plugin-rules/code-sample-checks/README.md:59]: There is a typo in sdk here.

File: custom-plugin-rules/code-sample-checks/check-sdk-coverage.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+// If you have different SDK languages edit this array
+const sdkLanguages = ['bash', 'javascript', 'python', 'ruby', 'java', 'kotlin'];
+
+function CheckSDKCoverage () {
+    return {
+        XCodeSampleList: {
+          enter(codeSampleList, ctx) {
+            // Make sure the list contains at least one bash sample
+            const hasBashSample = codeSampleList.some((codeSample) => {
+              return codeSample.lang === 'bash';
```
Comment (adamaltman) [inline custom-plugin-rules/code-sample-checks/check-sdk-coverage.js:12]: This is a bit weird -- is this a curl example? usually people would describe it as curl and it works with a variety of shells?

Comment (sanscontext) [reply custom-plugin-rules/code-sample-checks/check-sdk-coverage.js:12]: This is already merged, but yes. I think this was to do with the syntax highlighter on the render library we're using.

**Pull request merged**